openid-sharedsignals-framework-1_0.md

```diff
@@ -1229,6 +1229,13 @@ Errors are signaled with HTTP status codes as follows:
 {: title="Create Stream Errors" #tablecreatestream}
 
 
+##### Validating a Stream Creation Response
+
+* `aud`: It is RECOMMENDED that the Receiver validates the `aud` in the Create Stream Response.
+In most cases, a Transmitter and Receiver will agree upon the audience value out of band. 
+Regardless of how the Receiver obtains the audience, it SHOULD ensure that it matches the
```

Comment:
"obtains" doesn't sound right here. How about "Regardless of how the audience value is agreed upon"?